examples/web-wagmi/package.json

```diff
@@ -15,7 +16,7 @@
     "tsc": "tsc --noEmit"
   },
   "dependencies": {
-    "@bundlr-network/client": "^0.10.3",
+    "@bundlr-network/client": "^0.11.9",
```

Comment:
Had to upgrade bundler due to some heresy in build pipline.
https://github.com/vitejs/vite/issues/9703
Older `@bundlr-network/client` was using `near-api-js` which is probably wrongly published and was causing build failures for production `web-wagmi` deployment.